to-do-list/.env

```diff
@@ -0,0 +1,8 @@
+AUTH0_SECRET=
+AUTH0_BASE_URL=http://localhost:3000
+AUTH0_ISSUER_BASE_URL=
+AUTH0_CLIENT_ID=
+AUTH0_CLIENT_SECRET=
+AUTH0_REDIRECT_URI=http://localhost:3000/api/callback
+AUTH0_POST_LOGOUT_REDIRECT_URI=http://localhost:3000/
+AUTH_COOKIE_SECRET=
```

Comment:
Usually, the .env file is added to .gitignore file in order to prevent misconfigurations or typos, 

My advice here is to create a file called

`.env.sample`

To have the example of the file on the repo and add by ourselves in our local environment.

Reply:
I know but, I am not sure if it is because I have a config problem but my reactjs projects only read .env.local files. That is the reason why .env file isn't added to .gitignore and .env.local is. When I try to use .env I always get on process.env 'undefined'
🤷‍♂️

Reply:
Nevermind, I've read next.js doc (https://nextjs.org/docs/basic-features/environment-variables) and I will fix it.